log.go

```diff
@@ -148,6 +148,27 @@ func (l Level) String() string {
 	return ""
 }
 
+// LvlFromString returns the log level that corresponds to the supplied string value. The values are the same as returned by Level's String() method.
+func LvlFromString(l string) Level {
+	switch l {
+	case "debug":
+		return DebugLevel
+	case "info":
+		return InfoLevel
+	case "warn":
+		return WarnLevel
+	case "error":
+		return ErrorLevel
+	case "fatal":
+		return FatalLevel
+	case "panic":
+		return PanicLevel
+	case "":
+		return NoLevel
+	}
+	return NoLevel
```

Comment:
Shouldn't this case return an error instead?

Reply:
Assuming you're talking about the `return` statement - sure - I was just mirroring the other API. I can make it return an error instead.